app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivity.java

```diff
@@ -182,35 +182,35 @@ public void directionsApiCallBack(DirectionsResult result, List<Route> routeOpti
         setRoutesAdapter();
     }
 
-
     private void setRoutesAdapter() {
         // Android adapter for list view
         adapter = new RoutesAdapter(this, R.layout.list_routes, mViewModel.getRouteOptions());
         allRoutes.setAdapter(adapter);
         allRoutes.setOnItemClickListener(new AdapterView.OnItemClickListener() {
             @Override
             public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
-                if(carButton.isSelected() || transitButton.isSelected() || walkButton.isSelected()){
+                if (carButton.isSelected() || transitButton.isSelected() || walkButton.isSelected()) {
                     Intent openPaths = new Intent(RoutesActivity.this,
                             PathsActivity.class);
                     DirectionsRoute directionsResult = mViewModel.getDirectionsResult().routes[i];
-                    openPaths.putExtra("directionsResult", directionsResult);
+                    openPaths.putExtra("directionsResult", directionsResult);                    openPaths.putExtra("shuttle", true);
+                    openPaths.putExtra("shuttle", false);
                     startActivity(openPaths);
-                }
-                else{
-                    Toast.makeText(getApplicationContext(), "Paths view for Shuttle route has not been implemented yet.", Toast.LENGTH_SHORT).show();
-                    // TODO: Add shuttle route to map #51
-                    // Intent openPaths = new Intent(RoutesActivity.this,
-                    //         PathsActivity.class);
+                } else {
+                    if (mViewModel.getShuttles().isEmpty()) {
+                    Toast.makeText(getApplicationContext(), "Paths view for Shuttle route is not available if no shuttles exist.", Toast.LENGTH_SHORT).show();
+                    }
+                    Intent openPaths = new Intent(RoutesActivity.this,
+                            PathsActivity.class);
                     // DirectionsRoute directionsResult = mViewModel.getDirectionsResult().routes[i];
```

Comment:
do we still need that?

Reply:
I added it because if no shuttles are available (Sunday) we still get a clickable option saying "No shuttles available" so when the user clicks on that option this will make sure that the app doesn't crash